### PR TITLE
Platform free

### PR DESCRIPTION
**Pull request by boshtannik**

### CODE REVIEW

File: src/mesh_lib/node/packet/trait_implementations/serializer.rs [hunk 1]
```diff
@@ -95,11 +93,11 @@ impl Serializer for Packet {
         let mut data: PacketDataBytes = PacketDataBytes::new();
         for _ in 0..DATA_TYPE_SIZE {
             data.push(*bytes_iterator.next().unwrap_or_else(|| {
-                serial_debug!("Could not take byte for deserialization of data");
+                // serial_debug!("Could not take byte for deserialization of data");
                 &0u8
             }))
             .unwrap_or_else(|_| {
-                serial_debug!("Could not push byte of serialized data");
+                // serial_debug!("Could not push byte of serialized data");
```
Comment (boshtannik) [inline src/mesh_lib/node/packet/trait_implementations/serializer.rs:100]: Can such macrosses be defined in platform-serial crate, with no need to re-define them each time in any crate like platform-serial-arduino-nano, etc??

File: src/main.rs [hunk 2]
```diff
@@ -3,50 +3,51 @@
...
 
     init_timer(dp.TC0);
+    init_serial(default_serial!(dp, pins, 9600));
 
-    let mut mesh_node = init_node(NodeConfig {
-        device_identifier: 1 as AddressType,
-        listen_period: 150 as ms,
-        usart: default_serial!(dp, pins, 9600),
-    });
+    let mut mesh_node = Node::new(1 as AddressType, 150 as ms);
```
Comment (berkus) [inline src/main.rs]: this now looks less transparent - what is 1, what is 150 ms?

File: src/mesh_lib/node/packet/trait_implementations/serializer.rs (hunk 1 above)
Comment (berkus) [reply src/mesh_lib/node/packet/trait_implementations/serializer.rs:100]: use standard `log` macros? info! debug! etc

**Review by berkus — APPROVED** [review]

File: src/mesh_lib/node/packet/trait_implementations/serializer.rs (hunk 1 above)
Comment (boshtannik) [reply src/mesh_lib/node/packet/trait_implementations/serializer.rs:100]: @berkus If i just regular user, and I do have only one physical serial port on the board, how could i use this library and log? Like share one serial stream to debug and transfer data? Thankfully - the protocol is pretty stable to handle that case.

File: src/main.rs (hunk 2 above)
Comment (boshtannik) [reply src/main.rs]: @berkus Got it. LL turn it back

**Pull request merged**